### PR TITLE
Upgrade CAPI commands if from 1.8 to 2.1 first

### PR DESCRIPTION
**Pull request by mmcelroy91**
https://jira.d2iq.com/browse/D2IQ-88839

###Description
If a user has previously upgraded from 1.8 to 2.1, an upgrade to 2.2 without the following instructions will break cert-manager.

### Preview

You can preview the docs build using the following URL, adding the PR # where specified:
http://docs-d2iq-com-pr-4424.s3-website-us-west-2.amazonaws.com/

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

**Review by gvasquezvargas — APPROVED** [review]
LGTM! But we need to copy this changes over to the kommander upgrade docs before merging. Ping me and we can do this together if you want to @mmcelroy91 💟

**Comment by mmcelroy91** [comment]
fixed in https://github.com/mesosphere/dcos-docs-site/pull/4425 so closing this one

**Comment by mmcelroy91** [comment]
Not a duplicate after we decided these steps for upgrading CAPI components were NOT a workaround for Release Notes, but rather a step needed in both Essential and Enterprise.  This is the Enterprise section.

File: pages/dkp/konvoy/2.2/dkp-upgrade/upgrade-konvoy/enterprise/index.md [hunk 1]
```diff
@@ -52,24 +52,39 @@ New versions of DKP come pre-bundled with newer versions of CAPI, newer versions
 If you are running on more than one management cluster (Kommander cluster), you must upgrade the CAPI components on each of these clusters.
 
 <p class="message--warning"><strong>IMPORTANT:</strong>Ensure your <code>dkp</code> configuration references the management cluster where you want to run the upgrade by setting the <code>KUBECONFIG</code> environment variable, or using the <code>--kubeconfig</code> flag, <a href="https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/">in accordance with Kubernetes conventions</a>.
+    
 
-Run the following upgrade command for the CAPI components.
+1.  If your cluster was upgraded to 2.1 from 1.8, prepare the old cert-manager installation for removal:
```
Comment (joejulian) [inline pages/dkp/konvoy/2.2/dkp-upgrade/upgrade-konvoy/enterprise/index.md:57]:
```suggestion
1.  If your cluster was upgraded to 2.1 from 1.8, prepare the old cert-manager installation for upgrade:
```

File: pages/dkp/konvoy/2.2/dkp-upgrade/upgrade-konvoy/enterprise/index.md [hunk 2]
```diff
@@ -52,24 +52,39 @@ New versions of DKP come pre-bundled with newer versions of CAPI, newer versions
...
 
-The output resembles the following:
+1.  For <strong>all</strong> clusters, upgrade capi-components:
 
-```text
+    ```bash
+    dkp upgrade capi-components
+    ```
+
+1.  If your cluster was upgraded to 2.1 from 1.8, remove the old cert-manager you marked in step 1:
```
Comment (joejulian) [inline pages/dkp/konvoy/2.2/dkp-upgrade/upgrade-konvoy/enterprise/index.md:70]:
```suggestion
1.  If your cluster was upgraded to 2.1 from 1.8, remove the remaining old cert-manager resources from 1.8:
```

**Review by joejulian — CHANGES_REQUESTED** [review]

**Review by joejulian — APPROVED** [review]

**Pull request merged**